packages/fxa-shared/test/metrics/amplitude.js

```diff
@@ -318,13 +317,13 @@ describe('metrics/amplitude:', () => {
       let result;
 
       before(() => {
-        result = transform({ type: 'sms.ios' }, {});
+        result = transform({ type: 'pair.ios' }, {});
       });
 
       it('returned the correct event data', () => {
         assert.equal(result.event_type, 'fxa_connect_device - cadEvent');
         assert.deepEqual(result.event_properties, {
-          connect_device_flow: 'sms',
+          connect_device_flow: 'pairing',
```

Comment:
Just out of curiousity, are these flow names/ids defined or documented somewhere?

Reply:
not all of them. I'd love to get some docs sometime!